easybuild/tools/modules.py

```diff
@@ -469,6 +479,19 @@ def get_value_from_modulefile(self, mod_name, regex):
         else:
             raise EasyBuildError("Can't get value from a non-existing module %s", mod_name)
 
+    def get_conflicts_from_modfile(self,mod_name):
+        conflicts_re = re.compile('^conflict\s*(\S+).*$',re.M)
```

Comment:
does this also work for modules in Lua syntax?

please add unit tests for this...